### DISCUSSION TITLE
Fluid field total_fields not working inside "if" statement

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#2999] [Feature Requests]
**Description of the problem**
When checking if current 'count' value equals the 'total_fields' value in a Fluid field loop, I have to use this syntax:
`{if "{fluidfield:count}" == "{fluidfield:total_fields}"}YES{/if}`
instead of like this:
`{if fluidfield:count == fluidfield:total_fields}YES{/if}`

**Comment by intoeetive** [comment]
This is part of bigger problem of making variable modifiers available available inside conditionals
